Perspex.Controls/ToolTip.cs

```diff
@@ -116,8 +116,8 @@ private static void ShowToolTip(Control control)
                     ((ISetLogicalParent)popup).SetParent(control);
                 }
 
-                var cp = MouseDevice.Instance.GetPosition(control);
-                var position = control.PointToScreen(cp) + new Vector(0, 22);
+                var cp = MouseDevice.Instance?.GetPosition(control);
+                var position = control.PointToScreen(cp.HasValue ? cp.Value : new Point(0, 0)) + new Vector(0, 22);
```

Comment:
would this be better as:

    var cp = MouseDevice.Instance?.GetPosition(control) ?? new Point(0, 0);